lib/blazer.rb

```diff
@@ -50,6 +50,7 @@ class << self
     attr_accessor :check_schedules
     attr_accessor :mapbox_access_token
     attr_accessor :assignees
+    attr_accessor :assignees_with_email
```

Comment:
I think we can re-use the `assignees`, only change the Query -> to `select id, email from ....`